dbfit-java/oracle/src/main/java/dbfit/environment/OracleStoredProcedureCall.java

```diff
@@ -15,13 +15,13 @@
 
 import dbfit.util.Direction;
 
-public class OracleStoredProcedureCall extends DbStoredProcedureCall {
+public class OracleStoredProcedureCall extends DefaultDbStoredProcedureCall {
```

Comment:
I think we can keep the names as they are now for this PR. Maybe the class hierarchy and choice of inheritance needs revising but that's a separate story.